community/uboot-tools/APKBUILD

```diff
@@ -12,7 +12,7 @@ depends_dev=""
 makedepends="$depends_dev bison flex openssl-dev linux-headers sdl-dev"
 install=""
 subpackages=""
-source="ftp://ftp.denx.de/pub/u-boot/u-boot-$pkgver.tar.bz2"
+source="http://ftp.denx.de/pub/u-boot/$pkgname-$pkgver.tar.bz2"
```

Comment:
`$pkgname` is `uboot-tools` and this URL needs `u-boot`

Reply:
Yeah, it does not make sense to use $pkgname here. If we decide to rename the package, the url would still remain the same.